src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlSerializationReader.cs

```diff
@@ -805,12 +799,6 @@ protected byte[] ToByteArrayBase64(bool isNull)
             return ReadByteArray(true); //means use Base64
         }
 
-        /// <include file='doc\XmlSerializationReader.uex' path='docs/doc[@for="XmlSerializationReader.ToByteArrayHex"]/*' />
-        protected static byte[] ToByteArrayHex(string value)
-        {
-            return XmlCustomFormatter.ToByteArrayHex(value);
-        }
```

Comment:
It looks like these are protected members on a public class.  They're ok to remove I guess because they're not in the reference assembly?